### ISSUE TITLE
Weitzmann 1861: bad linebreaks, missing words in transcription

### ISSUE THREAD

**Issue by m-k94** [DCMLab/ddd#15]
I have not figured out the cause of this problem yet. But in the Weitzmann 1861 document, something seems to go seriously wrong with the automated transcription and its understanding of the text structure: 

![Bad Linebreaks - Issue 7-1-21](https://user-images.githubusercontent.com/75686579/104042715-1e1f7d80-51db-11eb-9010-57fedad796aa.jpg)

As you can see, the line structure of the paragraph is not at all well represented in the OCR transcription. Instead, it appears to add arbitrary line breaks after every few words, also missing several words in the process. On some pages, the transcribed text thus becomes almost illegible, to the point where I would rather transcribe it myself from scratch.

Could there be a way to fix this? Otherwise this document will take a lot more time. Maybe choosing a different print is an option? But I would still like to understand why this happens...

**Comment by fabianmoss** [comment]
The reason is probably the fraktur font and the OCR model that was used might be not optimized for that.
To fix it (if there are only few instances), you would have to go back to the segmentation view and correct the lines manually.

If this behavior persists for the entire book you are right in that it would be too time-consuming to do that for all lines. I will check if I get a better model for fraktur running.

**Comment by m-k94** [comment]
The problem appears to exist in other books as well, even ones that do not use fraktur font!

For example, I found it happening in Riemanns 'Ideen'. Page 1, the first half of the paragraph was very well transcribed, but then it suddenly started producing the same kind of gibberish as in the other Text. Since it's a modern print, the typeface cannot be the issue here. 

![Riemann Ideen p1 - Bad Linebreaks again](https://user-images.githubusercontent.com/75686579/104122803-84c1aa00-5347-11eb-9aae-11a09d98e417.png)

I now copied the text of that passage by hand, but it still relates to these fragments now... (I know it's probably bad practice, but I don't know how to redraw these things yet.)

The next two pages were perfectly fine, with all lines completely in order. Then, on page 4, it starts happening again. starting from 1-24. I did not fix this page yet, so that you can look. When you enter "segmentation mode", then "Layout" and click on the lines, you can see them highlighted on the page. It looks almost completely arbitrary to me.

Most likely we can fix it by learning more about Transkribus, for example, how to efficiently redraw and re-transcribe these (parent?) lines.

**Comment by m-k94** [comment]
I tried to solve this issue myself on the aforementioned page 4 in Riemann's Ideen, where only the last five lines were messed up. 

I basically first deleted all the bad lines and manually drew new ones with the "add baseline" button ("BL" Tool).
It is not necessary to add new lines first using the "L" Tool, since they get created automatically if you add a baseline and accept the pop up asking for a new parent line. It works pretty well actually. 

I then copied the transcribed lines by hand. If it only happens for a few lines every few pages, this seems like a pretty easy fix.
